tools/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -703,7 +703,7 @@ export class FluidPackageCheck {
                 continue;
             }
             const files = fs.readdirSync(dir, { withFileTypes: true });
-            if (files.some((dirent) => !dirent.isDirectory() && dirent.name.endsWith(".ts"))) {
+            if (files.some((dirent) => !dirent.isDirectory() && (dirent.name.endsWith(".ts") || dirent.name.endsWith(".spec.js")))) {
```

Comment:
there's a tsconfig but also spec.js files? weird

Reply:
Yeah if I was getting more directly involved I'd just convert them all to .ts, but there's a little more work involved there.  It could be another option.